README.md

```diff
@@ -1,5 +1,41 @@
 # falco-website
 
+## Prerequisites
+
+To build this website you need :
+
+* [hugo](https://gohugo.io/getting-started/installing/)
+* [nodejs](https://nodejs.org/en/download/)
+* [yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable)
+
+### Test & Build
+
+#### Clone repository
+
+```bash
+git clone git@github.com:falcosecurity/falco-website.git
+```
+
+#### Install theme
+
+```bash
+cd ./falco-website/themes/falco-fresh
+yarn install
+```
+
+#### Test in local
+
+```bash
+make server
+```
+You can access your local website on http://localhost:1313
+
+#### Build
```

Comment:
This command is only for publishing the site. It doesn't need to be included here.

Reply:
As it was already included in Makefile, I added it. Will remove it.